common/display/displayqueue.cpp

```diff
@@ -656,9 +666,7 @@ void DisplayQueue::HandleExit() {
   state_ |= kIgnoreIdleRefresh;
   power_mode_lock_.unlock();
   vblank_handler_->SetPowerMode(kOff);
-  if (!previous_plane_state_.empty()) {
-    display_->Disable(previous_plane_state_);
-  }
+  display_->Disable();
```

Comment:
Goal was that we dont even need to call this when display is disconnected right ?

Reply:
I mainly called to keep all the drm related calls in the DrmDisplay otherwise we need to move the dpms off property set to displayqueue.

Reply:
I mean you dont need to call it in this particular case. HandleExit can take a boolean based on which you can call this. If display is disconnected than we dont need to call it i.e. send false, in other cases call when if (!previous_plane_state_.empty())  is true

Reply:
you mean no need to call dpms  off during disconnect.

Reply:
ya

Reply:
then how display will be turned off if we don't call dpms off.

Reply:
When display is disconnected, it's already handled by kernel. We only need to handle it when power mode is off

Reply:
No kernel is not turning off the display. If I comment the disable part, after unplug also I see pipe/plane is on.

Reply:
hmm k than this change doesn't make much sense